src/ruby/puppetserver-lib/puppet/server/puppet_config.rb

```diff
@@ -53,12 +53,6 @@ def self.initialize_puppet(puppet_config)
 
     Puppet.settings.use :main, :master, :ssl, :metrics
 
-    Puppet::FileServing::Content.indirection.terminus_class = :file_server
-    Puppet::FileServing::Metadata.indirection.terminus_class = :file_server
-    Puppet::FileBucket::File.indirection.terminus_class = :file
-
-    Puppet::Node.indirection.cache_class = Puppet[:node_cache_terminus]
-
```

Comment:
Are we confident that these are totally unused prior to the first request coming in? And/or that the defaults are valid?

Reply:
I'm confident that this fixes some of the test failures seen in the match puppet [PR](https://github.com/puppetlabs/puppet/pull/7685), but I can't be sure there's not some case where these values must be set before any requests start coming in. I'd rather they not be duplicated though...maybe, after we fix the pipeline to be more stable, we can merge this and then see if there is anything else caught in testing?

Reply:
Works for me